ramls/requests-report.raml

```diff
@@ -0,0 +1,34 @@
+#%RAML 1.0
+title: Circulation
+version: v0.1
+protocols: [ HTTP, HTTPS ]
+baseUri: http://localhost:9130
+
+documentation:
+  - title: Circulation Business Logic API
+    content: <b>API for report generation</b>
+
+types:
+  requests: !include requests.json
+  error: !include raml-util/schemas/error.schema
+  errors: !include raml-util/schemas/errors.schema
```

Comment:
Do we return validation errors? The `collection-get` resource type does not indicate GET will, since validation errors are typically dealing with a JSON body. If we do return a 422, which is what the error and errors schemas would be used for, we should indicate that 422 is a possible response. Otherwise we should get rid of this.

Additionally, the `error` schema has a reference to `parameters`, so we would need to include that schema as well.

Reply:
Fixed